articles/step-by-step-on-how-to-user-nodejs-repl-tutorial/index.md

```diff
@@ -0,0 +1,220 @@
+### Introduction
+
+Node.js `Read-Evaluate-Print-Loop`(REPL) is an easy-to-use command-line tool for processing Node.js expressions.
+It captures the user's JavaScript code inputs, interprets and evaluates the result of this code, displays the result to the screen, and repeats the process till the user quits the shell.  
+It's also important to note that this tool does not require file creation to write codes. It comes with ready to use Node.js development environment.
+
+In this tutorial, we learn the basics of Node.js REPL, how we can use this amazing tool to run scripts without creating `.js` files.
+
+### Table of contents
+1. Introduction to REPL
+2. Basic Arithmetic operations using REPL
+3. Creating methods and variables using REPL
+
+### Prerequisites
+
+This tutorial requires:
+1. Node.js Installed in your development local machine.
+2. Basic knowledge of JavaScript and Node.js.
+
+### Objectives
+At the end of this tutorial, you should be able to use REPL tool to run your Node.js scripts.
+
+### Getting started with REPL
+
+Enter the following in the terminal:
+
+```bash
+node
+```
+
+**Output:**
+
+```bash
+> 
+```
+
+Entering the command `node` in the terminal starts the REPL command-line tool indicated by the `>` symbol.  
+This symbol acts as an indicator that JavaScript is ready to read and evaluate your code.  
+
+We can test the functionality by printing `hello world` using REPL as shown below:
+
+```bash
+node
+> let hello = 'hello world';
+undefined
+> hello
+'hello world'  // output
+> 
+```
+
+In the script, we assign the text `hello world` to a variable called `hello`. Now, on calling this variable `hello`, we get `hello world` as the output on the command shell. You also notice that on pressing ENTER, `undefined` is printed. This is because `hello` variable doesn't return a value.
```

Comment:
The explanation here is confusing.

> This is because `hello` variable doesn't return a value.

The `hello` variable does return a value. It returns the value: hello world. You have to mention that the declaration of the hello variable,

That is, the line:
```js
let hello = 'hello world';
```
does not return anything and hence returns `undefined`